01-supp-addressingData.Rmd

```diff
@@ -0,0 +1,168 @@
+---
+layout: page
+title: Programming with R
+subtitle: Addressing data
+minutes: 20
+---
+
+
+```{r, include = FALSE}
+source('tools/chunk-options.R')
+```
+
+> ## Objectives {.objectives}
+>
+> Understand the 3 different ways R can address data inside a data frame.
+>
+> Combine different methods for addressing data with the assignment operator to update subsets of data
+
+R is a powerful language for data manipulation. There are 3 main ways for addressing data inside R objects.
+
+* By index (slicing)
+* By logical vector
+* By name (columns and lists only)
+
+Lets start by loading some sample data:
+
+```{r readData}
+dat<-read.csv(file='data/sample.csv',header=TRUE, stringsAsFactors=FALSE)
```

Comment:
I added the stringsAsFactors=FALSE here because I really didn't want to worry about factors at the moment (I'm working on another supplemental lesson to address factors separately).  It becomes important for the final challenge. 
The text describing this command does comment on this, although I agree "don't worry about it..." may not be totally satisfactory.